cy_canvas/reducers/cyElements.js

```diff
@@ -3,7 +3,146 @@ import { unselectElement, getEdgesBetween,
   filterNode, getSourceId, getTargetId
 } from '../helper'
 
+Array.prototype.average = function() {
```

Comment:
I read a post saying something about "don't extend native class in js under any circumstance". I don't remember the exact reason but I do remember the author recommends to just use function for reusability.